python/lsst/skymap/tractInfo.py

```diff
@@ -309,9 +369,11 @@ def getOuterSkyPolygon(self):
         return makeSkyPolygonFromBBox(bbox=self.getBBox(), wcs=self.getWcs())
 
     def getWcs(self):
-        """Get WCS of tract
+        """Get WCS of tract.
 
-        @warning: this is not a deep copy
+        Notes
+        -----
+        **Warning:** this is not a deep copy
```

Comment:
SkyWcs is immutable, so this note is obsolete.

I suggest specifying the return type of `afw.math.SkyWcs`.